File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.java
import java.io.*;
import java.util.Arrays;
import java.util.Collections;
import java.util.StringTokenizer;

public class Main{
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        while (true) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            int[] a = new int[3];
            a[0] = Integer.parseInt(st.nextToken());
            a[1] = Integer.parseInt(st.nextToken());
            a[2] = Integer.parseInt(st.nextToken());
            Arrays.sort(a);

            if (a[0] == a[1] && a[1] == a[2] && a[2] == 0){
                break;
            }

            if (a[0] == a[1] && a[1] == a[2]){
                System.out.println("Equilateral");
            } else if (a[2] < a[0] + a[1]){
                if (a[0] == a[1] || a[1] == a[2]){
                    System.out.println("Isosceles");
                } else{
                    System.out.println("Scalene");
                }
            } else{
                System.out.println("Invalid");
            }
        }
    }
}

File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
answer=[]
while True:
    arr=list(map(int,input().split()))
    if arr==[0,0,0]:break
    arr.sort()
    if arr[0]+arr[1]<=arr[2]:answer.append("Invalid")
    elif arr[0]==arr[1]==arr[2]:answer.append("Equilateral")
    elif arr[0]==arr[1]or arr[1]==arr[2] or arr[0]==arr[2]:answer.append("Isosceles")
    else : answer.append("Scalene")
for i in answer:print(i)